pkg/reconciler/pullsubscription/pullsubscription.go

```diff
@@ -210,7 +210,7 @@ func (c *Reconciler) reconcile(ctx context.Context, source *v1alpha1.PullSubscri
 
 	source.Status.SubscriptionID = resources.GenerateSubscriptionName(source)
 
-	state, err := c.EnsureSubscriptionCreated(ctx, source, source.Spec.Project, source.Spec.Topic, source.Status.SubscriptionID)
+	state, err := c.EnsureSubscriptionCreated(ctx, source, source.Spec.Project, source.Spec.Topic, source.Status.SubscriptionID, *source.Spec.AckDeadline, source.Spec.RetainAckedMessages, *source.Spec.RetentionDuration)
```

Comment:
This is getting long enough that it should probably be a struct.

Reply:
https://github.com/GoogleCloudPlatform/cloud-run-events/issues/101